src/views/portal/Message.tsx

```diff
@@ -0,0 +1,169 @@
+								UCI.
+							</u>
+						</i>
+					</strong>
+				</p>
+
+				<h4>Communication</h4>
+				<p>
+					Communication during the event will primarily be done through Slack.
+					You will receive an invite via email closer to the event date.
+				</p>
+
+				<h4>Team Formation</h4>
+				<p>
+					Still looking for teammates? Attend our two team formation events,
+					Thursday (2/2) 7-8 PM in DBH 6011 and Friday (2/3) 7-8 PM in Student
+					Center Pacific Ballroom. You can also create an introductory slide{" "}
+					<a href="https://docs.google.com/presentation/d/1qjAzK9ocrI5Lg2D8l4VnE38CWDU2bW2v3XqAROA-vR4/edit#slide=id.g1ecf0247681_1_277">
+						here
+					</a>{" "}
+					to connect with other hackers or use the #team-formation channel in
+					Slack to find teammates! Teams are limited to groups of up to four.
+				</p>
+
+				<h4>Health and Safety Protocols</h4>
+				<p>
+					Please be mindful of your personal health and of those around you.
+					Although not required, we strongly encourage wearing masks, and please
+					stay home if you feel sick. To help us ensure your safety at the
+					venue, please keep your <strong>badge</strong> (issued at check in){" "}
+					<strong>visible at all times</strong>.
+				</p>
+
+				<h4>Venue Information</h4>
+				<p>
+					Hack at UCI 2023 will take place (not overnight) on Friday evening,
+					all-day Saturday, and approximately the first half of Sunday on-campus
+					at UC Irvine Student Center.{" "}
+					<strong className={styles.red_text}>
+						You must check in on all three event days in order to be eligible to
+						win prizes, specifically attending the opening ceremony on Friday,
+						project expo for judging on Sunday, and completing the Midway
+						check-in form on Saturday
+					</strong>
+					. Hackathon events will take place in the following rooms:
+				</p>
+				<p>
+					<strong>Pacific Ballroom Lobby:</strong> Check In
+					<br />
+					<strong>Pacific Ballroom ABCD:</strong> Hacking Space
+					<br />
+					<strong>Woods Cove B + C:</strong> Hacking Space
+					<br />
+					<strong>Moss Cove A + B:</strong> Workshops and Socials
+				</p>
+				<p>
+					Map:{" "}
+					<a href="https://map.uci.edu/?id=463#!m/255880">
+						https://map.uci.edu/?id=463#!m/255880
+					</a>
+				</p>
+				<p>
+					Floor Map:{" "}
+					<a href="https://conference.studentaffairs.uci.edu/node/24">
+						https://conference.studentaffairs.uci.edu/node/24
+					</a>
+				</p>
+				<h4>On-Campus Parking</h4>
+				<p>
+					<strong>Attendees will be responsible for their own parking.</strong>{" "}
+					The closest parking structure to our event is the UCI Student Center
+					Parking Structure. This structure provides both pay-by-space ($2 per
+					hour) and permit dispensers ($1 per hour, $13 for a day). You may
+					purchase a permit at any information kiosk, dispenser, or through
+					UCI&#39;s ParkbyPlate signage located in select parking areas. All
```

Comment:
Nitpick [repeating]: although `&#39;` works here, `&apos;` is probably more meaningful to a developer.